boards/arm/96b_ivy5661/doc/96b_ivy5661.rst

```diff
@@ -0,0 +1,110 @@
+.. _96b_ivy5661:
+
+96Boards IVY5661
+################
+
+Overview
+********
+
+96Boards IVY5661 board is based on the Unisoc uwp5661 soc chip. Zephyr
+applications use the 96b_ivy5661 configuration to run on these boards.
```

Comment:
The overview should contain a description of where to go to find out how to get this board.

e.g. vendor homepage.

Reply:
Thanks for the comment, we are preparing the web page of how to get this board, will update when it is ready.